### PR TITLE
fixes clock prosel. not working on all walls/floors

### LINKED ISSUE #18162: Clockwork proselytizer cannot proselytize subtypes of floors, such as white medbay flooring
[Note]: # (State what the issue is from a "whats wrong" prospective. Issue reports should clearly allow maintainers to understand whats wrong and how to test/reproduce if that is not obvious. Avoid ambiguity. Start your issue report below both of these lines (or remove them))

[Admins]: # (If you are reporting a bug that occured AFTER you used varedit/admin buttons to alter an object out of normal operating conditions, please verify that you can re-create the bug without the varedit usage/admin buttons before reporting the issue.)
@Xhuis

### PR DESCRIPTION
**Pull request by Iamgoofball**
Fixes #18162

### CODE REVIEW

**Comment by ChangelingRain** [comment]
This will conflict with Xhuis's work.

**Comment by Iamgoofball** [comment]
too bad he didn't claim the issue

File: code/game/gamemodes/clock_cult/clock_objects.dm [hunk 1]
```diff
@@ -556,17 +556,16 @@
...
-			operation_time = 50
-			new_obj_type = /turf/closed/wall/clockwork
-			alloy_cost = 10
-			valid_target = TRUE //Need to change valid_target to 1 or TRUE in each check so that it doesn't return an invalid value
-		if(/turf/open/floor/plating, /turf/open/floor/plasteel)
-			operation_time = 30
-			new_obj_type = /turf/open/floor/clockwork
-			alloy_cost = 5
-			valid_target = TRUE
+	if(istype(target, /turf/closed/wall))
```
Comment (ChangelingRain) [inline code/game/gamemodes/clock_cult/clock_objects.dm]: You need to exclude reinforced walls and cult walls, or at least I would heavily advise you do so.

Comment (Iamgoofball) [inline code/game/gamemodes/clock_cult/clock_objects.dm]: is there a reason for that

Comment (ChangelingRain) [inline code/game/gamemodes/clock_cult/clock_objects.dm]: Ghetto 'break through walls' tool for servants, also being able to convert enemy cult walls when they can't do so back seems a little silly

Comment (Iamgoofball) [inline code/game/gamemodes/clock_cult/clock_objects.dm]: just make cultists able to reconvert them, not my problem

**Comment by phil235** [comment]
Fix the title

File: code/game/gamemodes/clock_cult/clock_objects.dm (hunk 1 above)
Comment (ChangelingRain) [inline code/game/gamemodes/clock_cult/clock_objects.dm]: Oh, and you need to exclude the clockwork wall it converts to.

Comment (Iamgoofball) [inline code/game/gamemodes/clock_cult/clock_objects.dm]: that sounds good

**Comment by Xhuis** [comment]
Fixed in the next commit of #18085, close this

**Comment by Iamgoofball** [comment]
What the fuck, I coded this one myself

**Pull request closed without merging**